### DISCUSSION TITLE
没有上一版本好用

### DISCUSSION THREAD

**Discussion by niunaiyi** [jeessy2/ddns-go#668] [General]
简单直接，能看到刷新动作及结果。现在这一版，我作为一个程序员都不知道怎么用了。升完级，就是不去更新阿里云的DNS，看日志也看不出来什么。对了，我是在黑群晖里用的，不知道是不是套件封装人的问题。

**Comment by jeessy2** [comment]
我也没看出来你说得哪里不好用

**Reply by niunaiyi** [reply]
我记得上一版本，侧边栏有个日志，显示什么时候更新，更新结果。现在上面的日志点开，就一条日志。而且不更新，只是说某个端口。另外我配置好了的，上一版能工作的，但是更新后，阿里云的dns解析一直不变。

**Reply by jeessy2** [reply]
没看明白，截个图说明下
